### PR TITLE
Initialized Temporary Queue Warning

### PR DESCRIPTION
**Pull request by shubh128**
Resolves https://github.com/SimplQ/simplQ-frontend/issues/439

### CODE REVIEW

File: simplq/src/components/common/Popup/TempQueueWarning.jsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  root: {
+    width: '100%',
+    background: '#FF8000',
+    color: '#FFFFFF',
+    top: '0',
+    position: 'absolute',
+    zIndex: '100',
+  },
+}));
+
```
Comment (maaverik) [inline simplq/src/components/common/Popup/TempQueueWarning.jsx:19]: @shubh128, could you move this styling code to an scss file and import it for use. That's how it's done everywhere else, so let's keep things consistent.

**Review by maaverik — CHANGES_REQUESTED** [review]

File: simplq/src/components/common/Popup/TempQueueWarning.jsx (hunk 1 above)
Comment (maaverik) [reply simplq/src/components/common/Popup/TempQueueWarning.jsx:19]: If possible, try to reduce the usage of material UI. This is not a hard requirement, we just don't want to be too tightly coupled with any styling library. If this is too hard to do, leave it for now.

Comment (daltonfury42) [reply simplq/src/components/common/Popup/TempQueueWarning.jsx:19]: Let's keep it as a hard requirement here. Except for the `CloseIcon`, can you avoid all other material ui usage?

File: simplq/src/components/common/Popup/TempQueueWarning.jsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
+import React, { useState, useEffect } from 'react';
```
Comment (daltonfury42) [inline simplq/src/components/common/Popup/TempQueueWarning.jsx:1]: Move to a separate folder named `ribbon/` inside `common/` with two files,
```
/common/Ribbon/
  - index.jsx                          // for the component
  - ribbon.module.scss                 // for styles
``` 

You can use [LoginButton](https://github.com/shubh128/simplQ-frontend/tree/master/simplq/src/components/common/LoginButton) for refernce.

File: simplq/src/components/common/Popup/TempQueueWarning.jsx [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+import { isLoggedIn } from '../../../services/auth';
+
+const useStyles = makeStyles(() => ({
+  root: {
+    width: '100%',
+    background: '#FF8000',
+    color: '#FFFFFF',
+    top: '0',
+    position: 'absolute',
+    zIndex: '100',
```
Comment (daltonfury42) [inline simplq/src/components/common/Popup/TempQueueWarning.jsx:16]: Remove `z-index`, `position: 'absolute'` and `top: 0`.

Can you use flexbox?

File: simplq/src/components/pages/Admin/index.jsx [hunk 4]
```diff
@@ -94,6 +95,7 @@ export default (props) => {
   return (
     <div className={styles['admin-content']}>
       <HeaderSection />
+      <TempQueueWarning />
```
Comment (daltonfury42) [inline simplq/src/components/pages/Admin/index.jsx:98]: Lets call this component "Ribbon"

Rationale: We can reuse it multiple times with different content in future.

**Comment by daltonfury42** [comment]
Hi @shubh128 Have not heard back from you. Will you be finishing this?

**Pull request closed without merging**